packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -45,9 +45,16 @@ function PostStatus( { isOpened, onTogglePanel } ) {
 }
 
 export default compose( [
-	withSelect( ( select ) => ( {
-		isOpened: select( 'core/edit-post' ).isEditorPanelOpened( PANEL_NAME ),
-	} ) ),
+	withSelect( ( select ) => {
+		// We use isEditorPanelRemoved to hide the panel if it was programatically removed. We do
+		// not use isEditorPanelEnabled since this panel should not be disabled through the UI.
+		const { isEditorPanelRemoved, isEditorPanelOpened } = select( 'core/edit-post' );
+		return {
+			isRemoved: isEditorPanelRemoved( PANEL_NAME ),
+			isOpened: isEditorPanelOpened( PANEL_NAME ),
+		};
+	} ),
+	ifCondition( ( { isRemoved } ) => isRemoved ),
```

Comment:
This logic should be updated:

```suggestion
	ifCondition( ( { isRemoved } ) => ! isRemoved ),
```

Travis reports some failing tests and thy are valid.

Reply:
I can't push to your repository myself :(

I tried with the proposed change locally and it works properly.

Reply:
> I can't push to your repository myself

Likewise ;)

Great catch! Added that.